packages/block-library/src/navigation/edit/index.js

```diff
@@ -716,7 +753,9 @@ function Navigation( {
 								currentMenuId={ ref }
 								clientId={ clientId }
 								onSelect={ handleSelectNavigation }
-								onCreateNew={ resetToEmptyBlock }
+								onCreateNew={ () =>
+									createNavigationMenu( '', [] )
```

Comment:
At some point we should consider extracting a utility function which creates an empty Nav block. It would improve the code readability.